### PR TITLE
QOLOE-222 Footer component changes

### PR DESCRIPTION
**Pull request by SenChung**
Added the Footer changes for QOLOE-222

Note: I am checking the issue with -> Feedback submitted
           Missing feedback submitted message (not sure if its been developed or w.i.p)
This is controlled within Form IO and was working before.

### CODE REVIEW

File: src/components/bs5/footer/_colours.scss
Comment (ghazal-salehi) [inline src/components/bs5/footer/_colours.scss:1]: Can you please replace #414141 with --#{$prefix}color-default-color-light-text-default

File: src/components/bs5/footer/footer.scss
Comment (ghazal-salehi) [inline src/components/bs5/footer/footer.scss:1]: Can you please remove the commented out codes?

File: src/components/bs5/footer/_colours.scss
Comment (SenChung) [reply src/components/bs5/footer/_colours.scss:1]: Thanks @ghazal-salehi, I have updated this colour.

File: src/components/bs5/footer/footer.scss
Comment (SenChung) [reply src/components/bs5/footer/footer.scss:1]: thanks @ghazal-salehi, I have removed the comments

**Comment by Epinder-SSQ** [comment]
Missed changes from design QA

Headings "For government', 'Website feedback' and 'Cultural acknowledgement' font size:
-  xl, lg and md viewports: 20px / 1.25rem
-  sml and xs: 16px / 1rem

Padding:
-  Below radio button Heading: 20px / 1.25rem 
- Above and below separators" 20px / 1.25rem

**Comment by MitchAnderson93** [comment]
Does this conflict with changes:
https://github.com/qld-gov-au/qgds-bootstrap5/pull/300 ?

**Review by MitchAnderson93 — APPROVED** [review]

**Pull request merged**